src/defaultsettings.cpp

```diff
@@ -223,7 +225,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("desynchronize_mapblock_texture_animation", "true");
 	settings->setDefault("hud_hotbar_max_width", "1.0");
 	settings->setDefault("enable_local_map_saving", "false");
-	settings->setDefault("show_entity_selectionbox", "true");
+	settings->setDefault("show_entity_selectionbox", "false");
```

Comment:
I'd prefer for it to be in this PR, it's easy to change back so I don't think it'll be too much of a problem.
It would be really nice if there was an ingame changelog shown for new releases...

Reply:
This change is fine for me, and apparently for others too, seeing as the PR was merged.

Reply:
I'll be around to revert if there is a massive uproar over it. I'll be sneaking a Windows build with it to some people who play Minetest too